.travis/before-install-linux-python3.sh

```diff
@@ -4,7 +4,6 @@ set -euo pipefail
 
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
 sudo add-apt-repository ppa:fkrull/deadsnakes -y
```

Comment:
Perhaps this would work more reliably?

http://stackoverflow.com/a/37660551/7572003

```
addons:
  apt:
    sources:
      - deadsnakes # source required so it finds the package definition below
    packages:
      - python3.5
```